api/test/proxy-implementations/proxy-tracer.test.ts

```diff
@@ -34,6 +34,11 @@ describe('ProxyTracer', () => {
   let provider: ProxyTracerProvider;
   const sandbox = sinon.createSandbox();
 
+  //TODO replace with spy functions for shutdown and forceFlush
```

Comment:
Is this meant to be left TODO?

Reply:
No, I was thinking about adding unit tests here. In the end I decided against it as it would only retest the interface.